bindings/rust/s2n-tls-tokio/tests/common/mod.rs

```diff
@@ -5,7 +5,7 @@ use s2n_tls::{
     config,
     connection::Builder,
     error::Error,
-    security::{DEFAULT, DEFAULT_TLS13},
+    security::{self, DEFAULT_TLS13},
```

Comment:
Nit: why use self instead of just importing TESTING_TLS12?

Reply:
`TESTING_TLS12` can be anything so adding the context of `security` improves readability imo. However, since its used with `set_security_policy(&TESTING_TLS12)?;`, TESTING_TLS12 works just fine.